<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectScreen</class>
 <widget class="QDialog" name="ObjectScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo Objeto</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lbl_desc">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Digite as informações do novo objeto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_object_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome do objeto:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coord">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Coordenadas:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_object_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string extracomment="Testes"/>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_coord">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">Testes</string>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
